pyproject.toml

```diff
@@ -99,7 +99,7 @@ test = [
 ]
 docs = [
     "pyhf[xmlio,contrib]",
-    "sphinx>=7.0.0",  # c.f. https://github.com/scikit-hep/pyhf/pull/2271
+    "sphinx>=7.0.0,!=9.0.1,!=9.0.2",  # c.f. https://github.com/scikit-hep/pyhf/pull/2271, https://github.com/scikit-hep/pyhf/pull/2642
```

Comment:
Oh, we should migrate this out if still needed, else this will be confusing to find in the future.

Reply:
Done in PR https://github.com/scikit-hep/pyhf/pull/2645